### DISCUSSION TITLE
Does Bun hardlink on Windows?

### DISCUSSION THREAD

**Discussion by UltimateGG** [oven-sh/bun#10071] [Q&A]
One of the features I was excited for is taking up a lot less storage by just pointing the files to the cache so they only exist once on disk.
The documentation says

`Once a package is downloaded into the cache, Bun still needs to copy those files into node_modules. Bun uses the fastest syscalls available to perform this task. On Linux, it uses hardlinks; on macOS, it uses clonefile.`

But it does not specify for Windows. Does it just copy like npm?

**Comment by kpietraszko** [comment]
Answered here: https://github.com/oven-sh/bun/issues/10327#issuecomment-2061866021
